massgov/mayflower
# DP-9573 Remove raw filter from _patterns/02-molecules/event-teaser.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from _patterns/02-molecules/event-teaser.twig.

Template use in Mayflower
```
 ~/MassIT/mayflower/ [feature/DP-9573_molecules-event-teaser] grep -r 'event-teaser.twig' patternlab/styleguide/source
patternlab/styleguide/source/_patterns/03-organisms/by-author/event-listing.twig:              {% include "@molecules/event-teaser.twig" %}
patternlab/styleguide/source/_patterns/03-organisms/by-template/page-header-for-event-public-meeting.md:  * @organisms/by-author/event-teaser.twig
patternlab/styleguide/source/_patterns/03-organisms/by-template/page-header-for-event.md:  * @organisms/by-author/event-teaser.twig
patternlab/styleguide/source/_patterns/03-organisms/by-template/page-header~for-event-public-meeting.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/03-organisms/by-template/page-header~for-event.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/04-templates/01-content-types/event.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/05-pages/event.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/05-pages/event~end-date-with-time.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/05-pages/event~end-date.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/05-pages/event~public-meeting-agenda.json:        "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/05-pages/event~public-meeting-minutes.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/05-pages/event~public-meeting.json:      "path": "@molecules/event-teaser.twig",
patternlab/styleguide/source/_patterns/05-pages/event~recurring.json:      "path": "@molecules/event-teaser.twig",
 ~/MassIT/mayflower/ [feature/DP-9573_molecules-event-teaser]
```

## Related Issue / Ticket

- [DP-9573](https://jira.mass.gov/browse/DP-9573)


## Steps to Test


1.  Go to event page sample(s) and check for display issue at the event teaser area. 

## Screenshots
<img width="634" alt="dp-9573_event-event-teaser1" src="https://user-images.githubusercontent.com/9633303/43097710-eaba7780-8e8a-11e8-80a6-0e9e2a9fdbfa.png">



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
